person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  reader_attr :id
+  accessor_attr :name, :age
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
```

Comment:
-[ ] Your constructor parameter should be `name`, `age`, `parent_permission`. Kindly remove `id` from your initialize as it is not required